api/src/i18n/services/translation.service.ts

```diff
@@ -77,11 +77,31 @@ export class TranslationService extends BaseService<Translation> {
 
           switch (settingType) {
             case SettingType.multiple_text:
-              strings = strings.concat(value);
+              if (Array.isArray(value)) {
+                strings = strings.concat(value);
+              } else if (typeof value === 'string') {
+                this.logger.warn(
+                  `The plugin ${plugin?.name} setting '${key}' is incompatible with the settings.ts`,
+                );
+                this.logger.warn(
+                  `Expected type "array" received type "string"`,
+                );
+                strings = strings.concat([value]);
+              }
```

Comment:
Let's add an else and issue a warning if the type is different from the setting type.